ci/github-script/labels.js

```diff
@@ -172,6 +172,40 @@ module.exports = async function ({ github, context, core, dry }) {
     return prLabels
   }
 
+  async function handleAutoClose(item) {
+    const issue_number = item.number
+
+    if (item.labels.some(({ name }) => name === '0.kind: packaging request')) {
+      const body = [
+        'Thanks for opening this package request. Unfortunately, in order to mitigate the unsustainable growth of unmaintained packages, Nixpkgs is no longer using package requests to coordinate new package additions.',
+        '',
+        'That said, **we encourage you to [contribute](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#quick-start-to-adding-a-package) this package as a Pull Request yourself** - should you need any help during packaging, feel free to reach out on [Matrix](https://matrix.to/#/#dev:nixos.org) or [Discourse](https://discourse.nixos.org/c/dev/14). You are also welcome to open custom _Tracking_ issues if you would like the assistance of other volunteers in packaging a software ecosystem.',
+        '',
+        '*This change has the benefit of decreasing future maintainer burden on the long term, and was implemented after several contributors reached a consensus. You can read more about this on Pull Request [391926](https://www.github.com/NixOS/nixpkgs/issues/391926) and Issue [425040](https://www.github.com/NixOS/nixpkgs/issues/425040).*',
```

Comment:
I don't see any dynamic interpolation (etc). Would it be more maintainable to store this text in a dedicated file?

We already need a `ci` checkout to have access to this script, anyway.

Reply:
Also: where possible I prefer to use reference-style links. As per https://github.com/NixOS/nixpkgs/pull/434609#discussion_r2288910748

Also: some of this phrasing is the same as #434609. If my nit-picks are accepted there, then the wording here should be kept in sync 😅